plat/renesas/rcar/bl31_rcar_setup.c

```diff
@@ -176,7 +176,7 @@ void bl31_platform_setup(void)
 	/* Get the mpidr for boot cpu */
 	rcar_boot_mpidr = read_mpidr_el1() & 0x0000ffffU;
 
-	NOTICE("SCPI: installed handler, implementation level: 000000\n");
+	NOTICE("SCPI: installed handler, implementation level: 100000\n");
```

Comment:
What does this change mean?

Reply:
I have borrowed this from the origin patch series. I assume it is something like a version of software,  where each digit decodes implementation of the corresponding protocol part.